setup.cfg

```diff
@@ -2,7 +2,7 @@
 universal=1
 
 [codespell]
-skip = *.json,*.cpp,*.c,.riot,.tox,.mypy_cache,.git,*ddtrace/vendor
+skip = *.json,*.cpp,*.c,.riot,.tox,.mypy_cache,.git,*ddtrace/vendor,tests/settings/test_levhenstein_distance.py
```

Comment:
why do we need to skip the whole file here?

Reply:
The codespell has an errror on of of the test input, and from a quick look it looks like codespell doesn't support inline ignore https://github.com/codespell-project/codespell/issues/1212